src/js/views/globals/search/patient-search.scss

```diff
@@ -15,16 +15,41 @@
   }
 }
 
+.patient-search__clear-type-button {
```

Comment:
can probably be genericized with `patient-search__search-by-button` and margin could be set in the html with a `u-m...` class